### PR TITLE
Disable button on chrome internal URLs and while loading

### PR DESCRIPTION
**Pull request by pastak**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -96,11 +96,43 @@ function onClickHandler (info, tab) {
...
+    disableButton(tabId)
+  } else if (changeInfo.status === 'complete') {
+    chrome.tabs.get(tabId, function (tab) {
+      if (tab.url.match(/^https?:/)) {
+        enableButton(tab.id)
+        chrome.tabs.executeScript(tab.id, {
+          file: './content.js'
+        }, function () {})
+      } else {
+        disableButton(tab.id)
```
Comment (rakusai) [inline main.js]: Do not need this?

File: main.js [hunk 2]
```diff
@@ -96,11 +96,43 @@ function onClickHandler (info, tab) {
...
+  })
+})
+
 chrome.tabs.onUpdated.addListener(function (tabId, changeInfo) {
-  if (changeInfo.status === 'complete') {
-    chrome.tabs.executeScript(tabId, {
-      file: './content.js'
-    }, function () {})
+  if (changeInfo.status === 'loading') {
+    disableButton(tabId)
```
Comment (rakusai) [inline main.js]: Do not need this?

File: main.js (hunk 1 above)
Comment (pastak) [inline main.js]: We can delete it

**Comment by rakusai** [comment]
perfect!!

**Pull request merged**